deploy/docker/backend.Dockerfile

```
@@ -7,10 +7,12 @@ RUN cargo fetch --locked
 COPY backend/ ./
 RUN cargo build --release --target x86_64-unknown-linux-musl
 
-FROM gcr.io/distroless/static:nonroot
+FROM alpine:3.18 AS runtime
+RUN apk add --no-cache curl
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)